sirepo/package_data/static/js/sirepo.js

```diff
@@ -2160,6 +2165,20 @@ SIREPO.app.factory('requestSender', function(cookieService, errorService, userAg
         $window.open(self.formatUrl(routeName, params), '_blank');
     };
 
+    self.openSimulation = (app, route, simId) => {
+        const m = {};
+        m[route] = routeMapLocal(route, `/${route}/:simulationId`);
```

Comment:
Look up in schema instead

Reply:
Update: `/simulation-schema` works with the ajax call but not with sendRequest. In the latter case the flask request contains no form data. Other routes like `simulation-list` appear to work. Not sure where or why the data is getting lost...

Reply:
Need more info here. Is the ajax a normal form post and sendRequest is json post?

Reply:
I see the problem: simulation-schema and the corresponding API are designed to use form data through ajax. I think a new route and api are the way to go then.

Reply:
Latest commit adds simulationRoute which gets a specific local route for an app. I don't think we need the whole schema.

Not sure about the proper Spec or the naming; 'route' is used a lot. Maybe 'appSection' is better.